source/articles/show.html.slim

```diff
@@ -25,7 +25,7 @@
       .article__text= Markdown.new(article.body).to_html
 
     .article__questions
-      - if config[:staging]
+      - if config[:is_staging]
```

Comment:
Will we remove these alternative URLs once proper staging/production names configured?